editor/plugins/script_text_editor.cpp

```diff
@@ -167,6 +167,13 @@ void ScriptTextEditor::_load_theme_settings() {
 
 	text_edit->add_constant_override("line_spacing", EDITOR_DEF("text_editor/theme/line_spacing", 4));
 
+	TextEdit *tx = code_editor->get_text_edit();
+	tx->set_syntax_coloring(EditorSettings::get_singleton()->get("text_editor/highlighting/syntax_highlighting"));
+	tx->add_color_override("keyword_color", keyword_color);
+	tx->add_color_override("function_color", function_color);
+	tx->add_color_override("basetype_color", basetype_color);
+	tx->add_color_override("member_variable_color", member_variable_color);
```

Comment:
Settings should be set [here.](https://github.com/godotengine/godot/blob/master/editor/code_editor.cpp#L741)

`member_variable_color` and `function_color` colors are already being passed in [here](https://github.com/godotengine/godot/blob/master/editor/plugins/script_text_editor.cpp#L160) so no need to add them here :).